doc_src/en/OmegaT5_ProjectFolder.xml

```diff
@@ -4,143 +4,159 @@
+	project.</para>
+
+	<note>
+	  <para>You can also create OmegaT projects either manually or with scripts by
+	  copying a set of prewritten files to a new folder:</para>
+	  
+	  <itemizedlist>
+		<listitem>
+		  <para>Any given folder that contains a valid
+		  <filename>omegat.project</filename> file will be recognized as a
+		  translation project by OmegaT (even if it later needs manual
+		  adjustments).</para>
+		</listitem>
+
+		<listitem>
+		  <para>If the folder is an empty repository and the
+		  <filename>omegat.project</filename> file contains remote repository
+		  information, the project will be recognized as a team project by
+		  OmegaT:</para>
+		  
+		  <programlisting>...
     &lt;external_command&gt;&lt;/external_command&gt;
     &lt;repositories&gt;
         &lt;repository type="git" url="https://URL/OF/THE/REMOTE/PROJECT/REPOSITORY"&gt;
             &lt;mapping local="/" repository="/"/&gt;
         &lt;/repository&gt;
     &lt;/repositories&gt;
-&lt;/project&gt;</programlisting>
+&lt;/project&gt;
+		  </programlisting>
 
-		<para>See the <link linkend="how.to.setup.team.project"
-		endterm="how.to.setup.team.project.title"/> how-to for details.</para>
-	  </listitem>
-	</itemizedlist>
-  </note>
+		  <para>See the <link linkend="how.to.setup.team.project"
+		  endterm="how.to.setup.team.project.title"/> how-to for details.</para>
+		</listitem>
+	  </itemizedlist>
+	</note>
+  </section>
+
+  <section>
+	<title>Minimal contents</title>
+
+	<para>A project is a folder containing, at a minimum, the following:</para>
+	
+	<variablelist>
+	  <varlistentry id="project.folder.omegat">
+		<term id="project.folder.omegat.title">omegat</term>
+		<listitem>
+		  <para>This is the project folder that always contains <link
+		  linkend="project.folder.project.save.tmx"
+		  endterm="project.folder.project.save.tmx.title"/>, the project
+		  translation memory, and <link linkend="project.folder.project.stats"
+		  endterm="project.folder.project.stats.title"/>, the statistics data file
+		  for the project.</para>
+
+		  <para>Other files will be added over the course of the translation.</para>
+
+		  <para>See <link linkend="project.folder.omegat.folder">below</link> for
+		  details.</para>
+		</listitem>
+	  </varlistentry>
+
+	  <varlistentry id="project.folder.omegat.project.file">
+		<term id="project.folder.omegat.project.file.title">omegat.project</term>
+		<listitem>
+		  <para>This file contains the project parameters defined in the <link
+		  linkend="dialogs.project.properties">project properties</link>: source and
+		  target languages, their respective tokenizers, the type of segmentation
+		  used, etc.</para>
+
+		  <para>It also acts as an <emphasis>address book</emphasis> that
+		  specifies where project resources are located.</para>
+
+		  <para>OmegaT also automatically creates an
+		  <filename>omegat.project.bak</filename> to restore your project
+		  settings in case of need.</para>
```

Comment:
`in case of need` can be improved by adding what is a case.

Reply:
The main use case is if OmegaT crashes while the file was locked, because in some OS (Windows, but maybe others) the original omegat.project may be empty if not properly closed after the end of your work. 
I let you decide whenever it is useful to explain this point in the documentation or not.

Reply:
"to restore your project settings in case of need"
I would say instead "to be sure OmegaT can open your project even if the original file is corrupted". The use case I describe in the previous message shows us that this has nothing to do with the contents of the file itself (it is independent on the fact that you changed options or not) but with the locking mechanism which can potentially corrupt the file.
But maybe I think like a developer, I let you decide which formulation is the most clear for an user.

Reply:
I thought about that and since it is Omegat that does the thing (unlike the TM backups), I decided that it was not necessary to give more details. But I'll add something.

Reply:
What do you call "the thing"? Creation of the backup file (and display a message if it is used) is made by OmegaT but it is in order to solve a potential problem with locking mechanism, and the locking mechanism is done by the OS, reason why its behaviour can change from one OS to another.
I simply meant that I never had the case where the file contains something wrong (such as an incomplete XML or a wrong character inside), the only use case was when the file looks completely empty because locking mechanism made it look like that.
And also I remember that yesterday somebody asks to explain the case, so did I. I don't know why the question is not here anymore, did you remove or rewrite one message?

Reply:
The "thing" is the operation on the backup file to restore the settings. In this case, OmegaT handles everything. In the TM case, it is the user who has to manipulate the backup data, rename it, etc.

I think the question (by @miurahr) has been removed because it was in the conversation that was solved by my correction, and GitHub does not display it anymore.

But I do understand what you wrote.

Reply:
OK I see. I had not understood the relation with the backups of project_save.tmx
Maybe the possibility also to manage them in the UI may be considered for the future (the more: we have the SuperTmxMerge window to display the differences, if needed). But that is out of topic.

Reply:
We need to put that in a RFE (if that's not already there)...